python-test/features/steps/control_plane_policies.py

```diff
@@ -202,21 +286,18 @@ def check_agent_log_for_policies(expected_message, time_to_wait, container_id, l
     while time_waiting < timeout:
         logs = get_orb_agent_logs(container_id)
         policies_have_expected_message = \
-            check_logs_contain_message_for_policies(logs, expected_message, list_agent_policies_id, considered_timestamp)
+            check_logs_contain_message_for_policies(logs, expected_message, list_agent_policies_id,
```

Comment:
yes... its because of PEP8, actually.

![image](https://user-images.githubusercontent.com/78241475/149001466-b732fdc5-512e-47aa-bf9a-bc31833750d7.png)